base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -322,13 +322,13 @@ impl ConsensusConstants {
             max_target_time: 1800,
             min_difficulty: 60_000.into(),
```

Comment:
I think this is supposed to be 60m?

```suggestion
            min_difficulty: 60_000_000.into(),
```

Reply:
or was it 6m in weatherwax?

Reply:
Pretty sure it's 60_000 in weatherwax 
I think somewhere along the lines weatherwax's values have changed

![image](https://user-images.githubusercontent.com/1057902/150297440-ea71a57b-129c-4dbd-8269-30fddd3505d9.png)

Reply:
I think those got altered during the validator node merge

Reply:
Yeah